openmp/libomptarget/include/Shared/APITypes.h

```diff
@@ -62,6 +62,11 @@ struct __tgt_target_table {
       *EntriesEnd; // End of the table with all the entries (non inclusive)
 };
 
+/// This struct contains a handle to a loaded binary in the plugin device.
+struct __tgt_device_binary {
+  uint64_t handle;
```

Comment:
I could change it to `uintptr_t`. My thinking is that we pretty much assert that this thing is 64-bit currently and the type being `uint64_t` reflects that. I don't use `void *` because it's supposed to be opaque so the user shouldn't be concerned with the fact  that it's a pointer.

Reply:
If we assume only pointers, make it void *.

Reply:
I think `uintptr_t` is fine, it's a common method to just hide opaque handles this way and the user doesn't need to think of it as a pointer.